pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/ArrayInitializationVerbosenessRule.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.codestyle;
+
+import net.sourceforge.pmd.lang.java.ast.ASTLocalVariableDeclaration;
+import net.sourceforge.pmd.lang.java.rule.AbstractJavaRule;
+
+public class ArrayInitializationVerbosenessRule extends AbstractJavaRule {
+
+    @Override
+    public Object visit(ASTLocalVariableDeclaration node, Object data) {
+
+        if (node.isArray() && node.getImage() != null) {
```

Comment:
`isArray()` is deprecated, and will be removed in PMD 7.0.0, you shouldn't rely on it

Reply:
Removed and replaced with placeholder for now